openedx/core/djangoapps/discussions/views.py

```diff
@@ -3,24 +3,19 @@
 """
 from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication
 from edx_rest_framework_extensions.auth.session.authentication import SessionAuthenticationAllowInactiveUser
-from lti_consumer.models import LtiConfiguration
+
 from opaque_keys.edx.keys import CourseKey
 from opaque_keys import InvalidKeyError
-from rest_framework import serializers
 from rest_framework.permissions import BasePermission
 from rest_framework.response import Response
 from rest_framework.views import APIView
+from rest_framework import serializers
 
 from common.djangoapps.student.roles import CourseStaffRole
-from lms.djangoapps.discussion.rest_api.serializers import DiscussionSettingsSerializer
-from openedx.core.djangoapps.django_comment_common.models import CourseDiscussionSettings
-from openedx.core.lib.api.authentication import BearerAuthenticationAllowInactiveUser
-from openedx.core.lib.courses import get_course_by_id
-from xmodule.modulestore.django import modulestore
 
-from .models import DEFAULT_PROVIDER_TYPE
+from openedx.core.lib.api.authentication import BearerAuthenticationAllowInactiveUser
```

Comment:
Shouldn't this go above in the previous import group (project-local, sibling of `comon...`?
instead of here with app-local imports.